gp-ant-task/src/main/java/com/ibm/g11n/pipeline/ant/BundleSet.java

```diff
@@ -33,7 +31,7 @@
  */
 public class BundleSet {
 
-    private ResourceType type = ResourceType.JAVA;
+    private String type = "JAVA";
```

Comment:
change constant to `DefaultResourceFilterProvider.Filter.JAVA.name()`?

Reply:
also , is this really an `id` and not a type?

Reply:
Previously we used ResourceType enum. But enum does not fit well with dynamic service registration. I agree we have better way to define this.

`id` vs. `type`. I think it's resource type ID. We already use the term type in Maven/Ant interface, I cannot change the term in Maven/Ant easily.

Reply:
I'm looking at https://github.com/IBM-Cloud/gp-java-tools/pull/93/files#diff-4199ca2c76f4388d7b95620e3e72b839R44 where `CSV-MULTI` is the ID, and `Type.MULTI` is the type.  I think it was confusing where in the code 'type' refers to the filter ID

Reply:
sorry, this class is within ant, so maybe not relevant.

Reply:
sorry, this is in the ant class, so probably OK